src/System.Net.Http.WinHttpHandler/tests/UnitTests/ClientCertificateHelper.cs

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+using Xunit;
+
+namespace System.Net.Http.WinHttpHandlerUnitTests
+{
+    public class ClientCertificateHelper
+    {
+        private readonly X509Certificate2 _cert_KeyUsageIncludesDigitalSignature_EKUIncludesClientAuth_PrivateKey =
+            new X509Certificate2(
+                Convert.FromBase64String(
+                  @"MIIKTgIBAzCCCgoGCSqGSIb3DQEHAaCCCfsEggn3MIIJ8zCCBgwGCSqGSIb3DQEHAaCCBf0EggX5
```

Comment:
@CIPop Where exactly in the SslStream tests?  I assume it isn't checkec into [master] yet?

I don't see the value in putting it into a file since it would still be Base64 anyways. Only advantage would be for possible reuse in other test classes/libraries and stored in the Common/tests folder tree.